docker-images/oncoanalyser/assets/run.sh

```diff
@@ -346,31 +520,45 @@ elif [[ ${mode} == 'wgts_existing_both' ]]; then
   nextflow_args='--mode manual --processes_include cuppa'
 fi
 
-if [[ ! -z ${resume_nextflow_dir:-} ]]; then
+if [[ -n "${resume_nextflow_dir:-}" ]]; then
   nextflow_args+=' -resume'
 fi
 
-upload_data() {
-  aws s3 sync \
-    --no-progress \
-    --no-follow-symlinks \
-    --exclude='software/*' \
-    --exclude='assets/*' \
-    --exclude='work/*' \
-    ./ ${output_dir}/
-}
+## END NEXFLOW ARGS ##
+
+## CREATE NEXTFLOW CONFIG ##
+nextflow_config_tmp="$( \
+  mktemp --suffix "nextflow.config"\
+)"
```

Comment:
I had been meaning to capture the Nextflow config file when uploading .nextflow/, .nextflow.log, and samplesheet.csv to give us better reproducibility for runs. Though I'm undecided whether this is excessive.

At least for now, can we use a stable filename (e.g. `nextflow.config`) and place it into the cwd?

Reply:
Updated in 7acc9c13da2d7887fd0065cbb5f7971d1f22d4dc to use filename _nextflow.config_ in current working directory.